sympy/printing/tests/test_repr.py

```diff
@@ -101,12 +101,11 @@ def test_Rational():
 
 
 def test_Float():
-    sT(Float('1.23', prec=3), "Float('1.22998', prec=3)")
```

Comment:
> it's better to always display in terms of binary precision 

Yes I agree.

My concern is that srepr printing has changed: before it was `Float('1.22998', prec=3)` now it is `Float('1.23', precision=13)`.  I agree that 13 is the right thing to print.  But why has displayed digits changed?  Its not necessarily bad; just not yet explained.